README.md

```diff
@@ -1,8 +1,115 @@
+
 # supported ![CI](https://github.com/stefanpenner/supported/workflows/CI/badge.svg)
 
 
+
+
 ## Usage
 
+
 ```sh
 npx supported <path/to/node_module>
+npx supported <[array/of/node_modules]>
+```
+
+### As a node module
+
+
+```js
+const { isAllInSupportWindow } = require('supported');
+const projectPaths = ['test/fixtures/unsupported-project', 'test/fixtures/supported-project' ];
+const jsonResult = await isAllInSupportWindow(projectPaths);
+const projectPaths_2 = 'test/fixtures/unsupported-project';
+const jsonResult_2 = await isAllInSupportWindow(projectPaths_2);
+```
+
+
+
+Developers can pass custom setupProjectFn which will return a object as below
+
+```js
```

Comment:
This example likely needs more polish as it is hard to follow.
If you are in a rush, maybe we remove it for now and add this back at a later time once there is more time to iterate on it.

Reply:
I updated the readme with minimum items. We will update the polished readme later.